spyderbat-event-forwarder/forwarder.go

```diff
@@ -43,57 +40,16 @@ var (
 const (
 	requestDelay    = 30 * time.Second // how long to wait between requests
 	minQueryOverlap = 5 * time.Minute  // always look back at least this far
+	maxQueryTime    = 15 * time.Minute // don't query more than this at once
```

Comment:
part of me feels like we should turn this into a potential flag?  maybe in the future...